src/rest.cpp

```diff
@@ -285,6 +288,206 @@ static bool rest_block_notxdetails(HTTPRequest* req, const std::string& strURIPa
     return rest_block(req, strURIPart, false);
 }
 
+static bool rest_filter_header(HTTPRequest* req, const std::string& strURIPart)
```

Comment:
Since this is all new code, can you adhere to style guide? if blocks with braces or on same lines, variable names in camel case, etc.